### PR TITLE
Add validadors for brazilian and mercosul license plates

### PR DESCRIPTION
**Pull request by marceloabk**
#112 
I found this issue and wanted to contribute to the repository for the first time. Speaking of which, excellent work guys.

When I started writing the tests, I saw that there was a small grammatical error in "a object" and I took the liberty to include the correction in this PR, but I can remove it if you think it is better.

Any mistakes that I may have made, or lack of standardization, or if something was missing, let me know and I'll be ready to correct.

### CODE REVIEW

File: src/utilities/licensePlate/index.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+const validBrazilianLicensePlateRegex = /^[a-zA-Z]{3}-?[0-9]{4}$/;
+
+export function isValid(licensePlate: string): boolean {
+  if (!licensePlate || typeof licensePlate !== 'string') return false;
+
+  const matchedLicensePlate =
+    validMercosulLicensePlateRegex.exec(licensePlate) || validBrazilianLicensePlateRegex.exec(licensePlate);
+  if (!matchedLicensePlate) return false;
+
+  return true;
```
Comment (rfoel) [inline src/utilities/licensePlate/index.ts:11]:
```suggestion
  return validMercosulLicensePlateRegex.test(licensePlate) || validBrazilianLicensePlateRegex.test(licensePlate);
```

File: src/utilities/licensePlate/index.test.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+// Tests created using the following links as reference:
+// https://help.returnpath.com/hc/en-us/articles/220560587-What-are-the-rules-for-email-address-syntax-
+// https://stackoverflow.com/questions/2049502/what-characters-are-allowed-in-an-email-address
```
Comment (rfoel) [inline src/utilities/licensePlate/index.test.ts:3]: Should this be here?

**Review by rfoel — CHANGES_REQUESTED** [review]
Nice work, I just pointed out some minor improvements. I'd suggest a license plate formatter together with this feature.

File: src/utilities/licensePlate/index.test.ts (hunk 2 above)
Comment (marceloabk) [reply src/utilities/licensePlate/index.test.ts:3]: Should not, my bad. I made the PR fallowing the email as a template, sorry.

**Comment by marceloabk** [comment]
I even thought of a formatter, but as new plates from Mercosul don't have a separator for example, I think the formatter would just leave the letters in uppercase, do you think it's worth?

Changing the subject a little bit, I'm trying to participate in the hacktoberfest event, and for my participation to count the repository has to be with the `hacktoberfest` topic or the PR with the `hacktoberfest-accepted` label.
So if the PR manages to get approved, would it be possible to add the  `hacktoberfest-accepted` label please?

[Link for the hacktobefest policy](https://hacktoberfest.digitalocean.com/hacktoberfest-update)

File: docs/utilities.md [hunk 3]
```diff
@@ -104,6 +104,16 @@ import { isValidLandlinePhone } from '@brazilian-utils/brazilian-utils';
 isValidLandlinePhone('1130000000'); // true
 ```
 
+## isValidLicensePlate
```
Comment (hyanmandian) [inline docs/utilities.md:107]: Maybe you forgot to add `pt-BR` docs.

**Review by hyanmandian — CHANGES_REQUESTED** [review]
Hello, @marceloabk nice job! I requested some changes, please look and tell me if it makes sense to you.

**Comment by hyanmandian** [comment]
> I even thought of a formatter, but as new plates from Mercosul don't have a separator for example, I think the formatter would just leave the letters in uppercase, do you think it's worth?
> 
> Changing the subject a little bit, I'm trying to participate in the hacktoberfest event, and for my participation to count the repository has to be with the `hacktoberfest` topic or the PR with the `hacktoberfest-accepted` label.
> So if the PR manages to get approved, would it be possible to add the `hacktoberfest-accepted` label please?
> 
> [Link for the hacktobefest policy](https://hacktoberfest.digitalocean.com/hacktoberfest-update)

I put `hacktoberfest` tag in this `MR`.

File: docs/utilities.md (hunk 3 above)
Comment (marceloabk) [reply docs/utilities.md:107]: Ill be doing this right now, didnt see that.

**Comment by marceloabk** [comment]
> I put `hacktoberfest` label in this `MR`.

Really appreciate, but the right label is `hacktoberfest-accepted`. (The `hacktoberfest` one is for de repo tag if you guys want, but does not need this one)

**Comment by marceloabk** [comment]
@hyanmandian is there anything else that I can do?

**Review by hyanmandian — APPROVED** [review]
Awesome job @marceloabk ! Sorry for the delay, I was on vacation. After I publish your feature, I will ping you to check if everything is right, ok? 

Thanks!

**Comment by hyanmandian** [comment]
@all-contributors please add @marceloabk for code, doc, and tests.

**Comment by hyanmandian** [comment]
I publish the new version `1.0.0-rc.12`. Can you check if everything is ok?

**Pull request merged**